pkg/secrets/secrets.go

```diff
@@ -80,3 +85,69 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 
 	return userDataSecret, nil
 }
+
+// Encrypt creates an ecnrypted block of text from a plaintext message using the given key
+func Encrypt(plainText string, key []byte) (string, error) {
```

Comment:
Why is `Encrypt()`part of the `secrets` package. AFAICT this has nothing to do with the private key secret. Just because you are using the same private key to encrypt and decrypt doesn't mean it belongs to this package.

Reply:
I do see your point. Addressed this by moving the encrypt and decrypt functions to their own `crypto` package